src/widget/qopengl/wspinny.cpp

```diff
@@ -0,0 +1,824 @@
+#include "widget/wspinny.h"
```

Comment:
```suggestion
#include "widget/qopengl/wspinny.h"
```
I guess this file is meant to be included here?

Reply:
it doesn't matter, "widget/wspinny.h" will include "widget/qopengl/wspinny.h" when MIXXX_USE_QOPENGL is defined.